api/services/sic_lookup_client.py

```diff
@@ -41,12 +42,24 @@ def __init__(self, data_path: str | None = None) -> None:
         # Initialise the SIC lookup service
         self.lookup_service = SICLookup(resolved_path)
 
+        # Log confirmation of data loading
+        logger.info(
+            "Loaded %d SIC lookup codes from %s",
+            self.get_sic_codes_count(),
+            resolved_path,
+        )
+
     def _get_default_path(self) -> str:
         """Get the default path to the SIC lookup data file.
 
         Returns:
             str: Path to the SIC lookup data file.
         """
+        # If environment variable is set, use that path
```

Comment:
I don't understand why we are treating the environment provided data paths as default? The get_lookup_client()/SICLookupClient() and get_rephrase_client()/SICRephraseClient() have a data_path parameter that is optional. 

Why don't we just set a datapath there (when environment variables are available) and leave the default_path to refer to the packaged data?  This way the default path is always the example files packaged with the code and the client can be initialised by passing a non-default path for data provided via environment variables.